2025-10-21 17:19:47,787 INFO    Thread-12 (_run_job):3226244 [wandb_init.py:setup_run_log_directory():705] Logging user logs to /home/wzseu/JJC_project/wls_33/wandb/run-20251021_171947-p8njxl9v/logs/debug.log
2025-10-21 17:19:47,787 INFO    Thread-12 (_run_job):3226244 [wandb_init.py:setup_run_log_directory():706] Logging internal logs to /home/wzseu/JJC_project/wls_33/wandb/run-20251021_171947-p8njxl9v/logs/debug-internal.log
2025-10-21 17:19:47,787 INFO    Thread-12 (_run_job):3226244 [wandb_init.py:init():832] calling init triggers
2025-10-21 17:19:47,787 INFO    Thread-12 (_run_job):3226244 [wandb_init.py:init():837] wandb.init called with sweep_config: {'batch_size': 16, 'bias_scale': 3, 'data_dir': 'data/windows_ieee33', 'device': 'cuda', 'epochs': 100, 'hidden': 256, 'lambda_op': 1, 'lambda_pf': 0.1, 'lambda_smooth': 0.001, 'lambda_wls_reg': 15, 'layers': 2, 'lr': 0.0005, 'nhead': 4, 'tag': 'W24', 'use_mask': False, 'w_temp_th': 0.01, 'w_temp_vm': 0.01}
config: {'batch_size': 16, 'bias_scale': 3, 'data_dir': 'data/windows_ieee33', 'device': 'cuda', 'epochs': 100, 'hidden': 256, 'lambda_op': 1, 'lambda_pf': 0.1, 'lambda_smooth': 0.001, 'lambda_wls_reg': 15, 'layers': 2, 'lr': 0.0005, 'nhead': 4, 'tag': 'W24', 'use_mask': False, 'w_temp_th': 0.01, 'w_temp_vm': 0.01, '_wandb': {}}
2025-10-21 17:26:54,333 INFO    Thread-12 (_run_job):3226244 [wandb_run.py:_finish():2275] finishing run a530029885-southeast-university/gm-se-sweep/p8njxl9v
2025-10-21 17:26:54,333 INFO    Thread-12 (_run_job):3226244 [wandb_run.py:_atexit_cleanup():2474] got exitcode: 0
2025-10-21 17:26:54,334 INFO    Thread-12 (_run_job):3226244 [wandb_run.py:_restore():2456] restore
2025-10-21 17:26:54,334 INFO    Thread-12 (_run_job):3226244 [wandb_run.py:_restore():2462] restore done
2025-10-21 17:26:56,343 INFO    Thread-12 (_run_job):3226244 [wandb_run.py:_footer_sync_info():3872] logging synced files
2025-10-21 17:27:06,348 INFO    wandb-AsyncioManager-main:3226244 [service_client.py:_forward_responses():80] Reached EOF.
2025-10-21 17:27:06,348 INFO    wandb-AsyncioManager-main:3226244 [mailbox.py:close():137] Closing mailbox, abandoning 1 handles.
